src/Clock.elm

```diff
@@ -0,0 +1,103 @@
+module Clock exposing (Model, Msg(..), init, main, subscriptions, update, view)
```

Comment:
Don't expose `Msg`'s constructors and `main` - they shouldn't be used externally